operator/controller/src/main/docker/Dockerfile.jvm

```diff
@@ -0,0 +1,19 @@
+FROM registry.access.redhat.com/ubi8/openjdk-17-runtime:latest
+
+ENV JAVA_APP_DIR=/deployments \
+    APP_URL="quarkus-app" \
+    JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager" \
```

Comment:
`-Dquarkus.http.host=0.0.0.0` can be removed

Reply:
Done.